seed/static/seed/js/services/geocode_service.js

```diff
@@ -6,7 +6,7 @@ angular.module('BE.seed.service.geocode', [])
       var geocode_factory = {};
 
       geocode_factory.geocode_by_ids = function (property_state_ids, taxlot_state_ids) {
-        return $http.post('/api/v2/geocode/geocode_by_ids/', {
+        return $http.post('/api/v3/geocode/geocode_by_ids/', {
           property_ids: property_state_ids,
           taxlot_ids: taxlot_state_ids
         }, {
```

Comment:
We changed the request payload to accept view ids rather than state ids to make the APIs more consistent. Happy to help with this if you have any questions as it requires a few changes on the FE